wp-content/plugins/civicrm/civicrm/ext/afform/core/ang/af/afForm.component.js

```
@@ -333,48 +333,53 @@
-            if (endDate < startDate) {
-              errorMessage += `Collections cannot end (${endDateValue}) before start (${startDateValue}).\n`;
-              isValid = false;
-            }
-          }
+        var startDateParts = startDateValue ? startDateValue.split('/') : institutionStartDateValue.split('/');
+        var endDateParts = endDateValue ? endDateValue.split('/') : institutionEndDateValue.split('/');
+        
+        var startDate = new Date(startDateParts[2], startDateParts[1] - 1, startDateParts[0]);
+        var endDate = new Date(endDateParts[2], endDateParts[1] - 1, endDateParts[0]);
+        
+        var errorMessage = '';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'errorMessage'. Consider to delete it or rename it.

'errorMessage' is defined here:


(lint/suspicious/noRedeclare)